src/snowflake/sqlalchemy/base.py

```diff
@@ -296,7 +295,7 @@ def visit_aws_bucket(self, aws_bucket, **kw):
             encryption_list.sort(key=operator.itemgetter(0))
         encryption = "ENCRYPTION=({})".format(
             " ".join(
-                ("{}='{}'" if isinstance(v, string_types) else "{}={}").format(n, v)
+                ("{}='{}'" if isinstance(v, str) else "{}={}").format(n, v)
```

Comment:
`string_types` no longer exists in SQLAlchemy 2.0. It used to be for Python 2 compatibility, but even in SQLAlchemy 1.4 it's just `string_types = (str,)`.